### ISSUE TITLE
User-viewable log

### ISSUE THREAD

**Issue by FFY00** [bimbo-garden/greenhouse#11]
As it is very common for people to have other users managing their channels, we should have an audit log where all actions are registered.

**Comment by IZeeLikeEyeZee** [comment]
Agreed, with an option to private it (to keep from people harassing mods with emote requests, for example hasan's emote editors are kept low-key and they would probably like to keep it that way)

**Comment by FFY00** [comment]
Yeah, this would be private.
